pallets/attestation/src/mock.rs

```diff
@@ -156,6 +156,7 @@ pub fn insert_attestation<T: Config>(claim_hash: ClaimHashOf<T>, details: Attest
 	kilt_support::reserve_deposit::<AccountIdOf<T>, CurrencyOf<T>>(
 		details.deposit.owner.clone(),
 		details.deposit.amount,
+		&T::RuntimeHoldReason::from(HoldReason::Deposit),
```

Comment:
We know the reason, right? I think we can simply avoid putting it here and simply use it inside the function.

Reply:
resolved in: [ed0fc93](https://github.com/KILTprotocol/kilt-node/pull/527/commits/ed0fc93380326ddaf001af7dc603c15ff552ae12)